soapui/src/main/java/com/eviware/soapui/impl/rest/discovery/DiscoveredRequest.java

```diff
@@ -0,0 +1,225 @@
+/*
+ *  SoapUI, copyright (C) 2004-2014 smartbear.com 
+ *
+ *  SoapUI is free software; you can redistribute it and/or modify it under the 
+ *  terms of version 2.1 of the GNU Lesser General Public License as published by 
+ *  the Free Software Foundation.
+ *
+ *  SoapUI is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without 
+ *  even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ *  See the GNU Lesser General Public License for more details at gnu.org.
+ */
+package com.eviware.soapui.impl.rest.discovery;
+
+import com.eviware.soapui.impl.rest.RestRequestInterface;
+import com.eviware.soapui.impl.wsdl.monitor.WsdlMonitorMessageExchange;
+import com.eviware.soapui.support.types.StringToStringsMap;
+import com.eviware.soapui.tools.PropertyExpansionRemover;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+import javax.mail.internet.ContentType;
+import javax.mail.internet.ParseException;
+import java.io.UnsupportedEncodingException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class DiscoveredRequest {
```

Comment:
Everything pertaining to Discovery should be removed. It won't work in SoapUI OS anyway.